src/openeo_test_suite/tests/workflows/test_load_apply_save.py

```diff
@@ -0,0 +1,34 @@
+from pathlib import Path
+import xarray as xr
+
+def test_load_apply_save_netcdf(connection,bounding_box_small,temporal_interval_one_day,s2_stac_url):
+    from openeo.processes import clip
+    filename = "test_load_apply_save_netcdf.nc"
+    Path(filename).unlink(missing_ok=True)
+    cube = connection.load_stac(
+        url = s2_stac_url,
+        spatial_extent = bounding_box_small,
+        temporal_extent = temporal_interval_one_day,
+        bands = ["B04"]
+    )
+    cube = cube.apply(lambda x: x.clip(0,1))
+    cube.download(filename)
+    assert Path(filename).exists()
+    assert (xr.open_dataarray(filename).max().item(0)) == 1
+
+
+def test_load_apply_dimension_save_netcdf(connection,bounding_box_small,temporal_interval_one_day,s2_stac_url,b_dim):
```

Comment:
consider enabling the pre-commit hook to automatically clean up the code formatting style : 
https://github.com/Open-EO/openeo-test-suite#pre-commit-hooks